docs/preprocessing.md

```diff
@@ -0,0 +1,13 @@
+## Executing fMRIPrep
+In order to run multiple sessions in parallel for a single subject, a trick needs to be applied to avoid several jobs to work simultaneously on the same file thus compromising it.
+
+- [ ] First, run only the anatomical workflow through a single fMRIPrep instance to fetch the needed template from templateflow and to generate the outputs of FreeSurfer. 
+``` bash title="Executing anatomical workflow of fMRIPrep"
+{% include 'code/fmriprep/ss-fmriprep-anatonly.sh' %}
+```
```

Comment:
```suggestion
    ``` bash title="Executing anatomical workflow of fMRIPrep"
    {% include 'code/fmriprep/ss-fmriprep-anatonly.sh' %}
    ```
```